README.md

```diff
@@ -1,92 +1,115 @@
+
+## Repository organization
+
+The `main` branch contains the latest state of development; do not use this for production. Refer to the [releases page](https://github.com/eosnetworkfoundation/mandel-contracts/releases) for current information on releases, pre-releases, and obsolete releases as well as the corresponding tags for those releases.
 ## Supported Operating Systems
 
-To speed up development, we're starting with **Ubuntu 20.04**. We'll support additional operating systems as time and personnel allow. In the mean time, they may break.
+[CDT](https://github.com/eosnetworkfoundation/mandel.cdt) is required to build contracts. Any operating systems supported by CDT is sufficient to build the system contracts.
 
+To build and run the tests as well, [EOSIO](https://github.com/eosnetworkfoundation/mandel) is also required as a dependency, which may have its further restrictions on supported operating systems.
 ## Building
 
-To speed up development and reduce support overhead, we're initially only supporting the following build approach. CMake options not listed here may break. Build scripts may break. Dockerfiles may break.
+The build guide below will assume you are running Ubuntu 20.04 though, as mentioned above, other operating systems may also be supported.
+
+### Build or install CDT dependency
+
+The CDT dependency is required. This release of the system contracts requires at least version 3.0 of CDT. 
+
+The easiest way to satisfy this dependency is to install CDT on your system through a package. Find the release of a compatible version of CDT from its [releases page](https://github.com/eosnetworkfoundation/mandel.cdt/releases), download the package file appropriate for your OS from the attached assets, and install the package.
+
+Alternatively, you can build CDT from source. Please refer to the guide in the [CDT README](https://github.com/eosnetworkfoundation/mandel.cdt#building) for instructions on how to do this. If you choose to go with building CDT from source, please keep the path to the build directory in the shell environment variable `CDT_BUILD_PATH` for later use when building the system contracts.
+
+### Build or install EOSIO dependency
+
+The EOSIO dependency is optional. It is only needed if you wish to also build the tests using the `BUILD_TESTS` CMake flag.
 
-### Ubuntu 20.04 dependencies
+The easiest way to satisfy this dependency is to install EOSIO on your system through a package. Find the release of a compatible version of EOSIO from its [releases page](https://github.com/eosnetworkfoundation/mandel/releases), download the package file appropriate for your OS from the attached assets, and install the package.
```

Comment:
As of right now we don't support the dev parts in an externally available package.  So I would remove the section about installing from a package.

Reply:
Oh right. Totally forgot about that. Thanks!